code/game/machinery/turrets.dm

```diff
@@ -87,8 +87,43 @@
 	anchored = 1
 	layer = 3.5
 	density = 0
+	var/maintenance = 0
+	var/cable_cut = 0
 	var/obj/machinery/turret/host = null
 
+/obj/machinery/turretcover/attackby(obj/item/I, mob/user)
+	if(!host.enabled)
+		if(istype(I, /obj/item/weapon/screwdriver))
+			if(maintenance)
+				playsound(loc, 'sound/items/Screwdriver.ogg', 100, 1)
+				user << "<span class='notice'>You close the covers panel.</span>"
+				maintenance = 0
+				return
+			else if(!maintenance)
+				playsound(loc, 'sound/items/Screwdriver.ogg', 100, 1)
+				user << "<span class='notice'>You open the covers panel.</span>"
+				maintenance = 1
+				return
+		if(maintenance)
+			if(istype(I, /obj/item/weapon/wirecutters))
+				if(host.health != 80 && !cable_cut)
+					playsound(loc, 'sound/items/Wirecutter.ogg', 100, 1)
+					user << "<span class='notice'>You remove the damaged wires.</span>"
+					cable_cut = 1
+			if(istype(I, /obj/item/stack/cable_coil))
+				var/obj/item/stack/cable_coil/coil = I
+				if(host.health != 80 && cable_cut)
+					coil.use(1)
+					host.health += 10
+					if(host.health < 80)
+						user << "<span class='notice'>You fix some of the turrets damage.</span>"
+					if(host.health >= 80)
+						user << "<span class='notice'>You fully repair the turret.</span>"
+						host.health = 80
+						cable_cut = 0
+	else

```

Comment:
remove this else and just have it parent call if the proc hasnt returned by then. place returns on all the if statements